### ISSUE TITLE
Zee5 not playing next video

### ISSUE THREAD

**Issue by denesemattlincbex** [Vikassm73/AjaykRepo#261]
Hi,

I tried to play next TV shows episodes in zee5, but they arranged in reverse order and video was still continuously playing in the end after the playback stopped

The issue was not observed in hotstar and Jio cinema add-ons, they are playing next episodes fine.

This issue can be reproduced by simply playing any free TV shows episodes in zee5, jump to the end of the video.

File location: Zips/plugin.video.zee5/plugin.video.zee5-2.1.24.zip

Checked already open issue but none of them related to this issue, so I opened this new issue.
![it](https://github.com/Vikassm73/AjaykRepo/assets/158460957/b1b13b2a-5583-4741-b9af-c2896d1fce3f)
![it2](https://github.com/Vikassm73/AjaykRepo/assets/158460957/a463ae0a-43ee-45c9-bb40-48590bb31443)

Thanks in advance for your great work making the add-ons working. It was so helpful for everyone.

Any help here highly appreciated 👍

**Comment by Vikassm73** [comment]
If you select stream 2, it is working fine. Stream 1 has an issue. I will check this. (If you are not getting stream selection then you have to change stream selection type in setting)

**Comment by denesemattlincbex** [comment]
Thanks for your response @Vikassm73 
I tried with stream 2, and it's working fine, but we should be selecting stream 2 each time before playing the video. Can we make this selection stream 2 by default?
The second problem with the Zee5 add-on is that episodes are arranged in descending order; for example, if I play episode number 143 by the end of the episode, it will play its next episode in the queue, which is episode number 142, and so on and so forth.
But episodes are perfectly arranged in ascending order in the Hotstar add-on, which is really helpful in playing the next video.
Is there any way we can change this order in the add-on itself?
As always, thanks for the prompt response.

**Comment by Vikassm73** [comment]
Sort order is default as received in a api. Actually descending order is best for a regular viewer. Those who has watched all previous episodes and just need to watch recent one.

**Comment by denesemattlincbex** [comment]
I agree on that point, but it's one button click to jump to a new episode in ascending order, too. For example, in hotstar you just need to press the ⬆️ up arrow button to jump to the latest episode instead of pressing the one-step ⬇️ down arrow button in the Zee5 add-on. But anyhow, I'm interested to know if we can change API output descending to ascending.
